__tests__/app.test.js

```diff
@@ -82,3 +82,24 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:id", () => {
+  test("200: responds with an individual article", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
```

Comment:
Are you expecting _any_ number? Or are you expecting the returned article to have a specific id?